sc/src/main/javascript/bundles/scxa-homepage-panels/html/index.html

```diff
@@ -24,15 +24,18 @@
 <!-- Set to http://localhost:8080/gxa/ or http://localhost:8080/gxa_sc/ -- Remember the trailing slash! -->
```

Comment:
This should not be part of this PR. Please remove it and all SC-related code and make a separate PR for changes related to Single Cell.